customers.sql

```diff
@@ -0,0 +1,8 @@
+select 
+o.customer_id, c.name, c.email, min(o.created_at) as first_order_at, count(o.id) as number_of_orders
+from `analytics-engineers-club.coffee_shop.customers` c
+join `analytics-engineers-club.coffee_shop.orders` o
+  on c.id = o.customer_id
```

Comment:
I like to specify my joins! I prefer to work in all left joins when I can -- to me it's the most logical way of joining together tables. Is this a LEFT RIGHT INNER or OUTER join?

Reply:
I'm used to only qualifying non-inner joins so that they stand out more. I fully recognize that it's a somewhat unpopular style preference, but it's one of the few that I hold strongly loll.